solr/solrj/src/java/org/apache/solr/client/solrj/request/GenericV2SolrRequest.java

```diff
@@ -40,8 +40,7 @@ public GenericV2SolrRequest(METHOD m, String path) {
    * @param params query parameter names and values for making this request.
    */
   public GenericV2SolrRequest(METHOD m, String path, SolrParams params) {
-    super(m, path);
-    this.params = params;
+    super(m, removeLeadingApiRoot(path), params);
```

Comment:
@gerlowskija this includes a minor bug fix (to call removeLeadingApiRoot as the other overloaded constructor does) but maybe this code wasn't used